src/applications/disability-benefits/526EZ/components/FormStartControls.jsx

```diff
@@ -2,28 +2,37 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import LoadingIndicator from '@department-of-veterans-affairs/formation/LoadingIndicator';
+
 import SaveInProgressIntro from '../../../../platform/forms/save-in-progress/SaveInProgressIntro';
+import backendServices from '../../../../platform/user/profile/constants/backendServices';
 
 import { ITFErrorAlert, VerifiedAlert, UnauthenticatedAlert, UnverifiedAlert, ITFDescription } from '../helpers';
 
+
 export default function FormStartControls(props) {
-  const { user } = props;
+  const { user: { login: { currentlyLoggedIn }, profile: { verified, services } } } = props;
+  const { EVSS_CLAIMS } = backendServices;
 
   const somethingWentWrong = props.ITFStatus && props.ITFStatus === 'expired' || // This may not be possible
                              props.errors && props.errors.length;
+  const hasEVSSClaimsService = !!(services.find(service => service === EVSS_CLAIMS));
+  const gateAuthenticatedUser = !verified && currentlyLoggedIn;
+  const gateVerifiedUser = !hasEVSSClaimsService && verified;
+  const permitVerifiedUser = verified && hasEVSSClaimsService;
   return (
     <div>
-      {!user.login.currentlyLoggedIn && <div>
+      {!currentlyLoggedIn && <div>
         {UnauthenticatedAlert}
         <button className="usa-button-primary" onClick={props.authenticate}>Sign In and Verify Your Identity</button>
       </div>}
-      {!user.profile.verified && user.login.currentlyLoggedIn && <div>
+      {gateAuthenticatedUser && <div>
         {UnverifiedAlert}
         <a href={`/verify?next=${window.location.pathname}`} className="usa-button-primary verify-link">Verify Your Identity</a>
       </div>}
+      {gateVerifiedUser && <div className="usa-alert usa-alert-error no-background-image">Sorry, our system is temporarily down while we fix a few things. Please try again later.</div>}
```

Comment:
So we'll display this alert if the user is logged in and LOA3, but does not have the `evss_claims` service, right?

@kreek Is not having `evss_claims` a temporary state, or will the user have to take some action to fix the problem?